grakn-test/test-unit/src/test/java/ai/grakn/test/engine/lock/LockTestIT.java

```diff
@@ -180,7 +180,7 @@ public void whenGettingLockWithDotInPath_LockIsAcquired(Locks locks){
 
     @Theory
     public void whenGettingLockWithDoubleDotInPath_LockIsAcquired(Locks locks){
-        String lockPath = "/..";
+        String lockPath = "/src/main";
```

Comment:
I think changing this path here invalidates the test. Look at the test title : `whenGettingLockWithDoubleDotInPath_LockIsAcquired `